x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/rules_table_toolbar.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiSwitch, EuiTab, EuiTabs, EuiToolTip } from '@elastic/eui';
+import React from 'react';
+import styled from 'styled-components';
+import { useRulesTableContext } from '../../../../containers/detection_engine/rules/rules_table/rules_table_context';
+import * as i18n from '../translations';
+
+const ToolbarLayout = styled.div`
+  display: grid;
+  grid-template-columns: 1fr auto;
+  align-items: center;
+  grid-gap: 16px;
+  box-shadow: inset 0 -1px 0 #d3dae6;
```

Comment:
can we use color from euiTheme?

Reply:
Yes, sure. I'll change that in a follow-up